k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -38,11 +38,37 @@ actions:
 - name: Execute CQL query
   bashTest:
     script: |-
-      set -x
-      export CQLSH_HOST=$APP_INSTANCE_NAME-cassandra-0.$APP_INSTANCE_NAME-cassandra-svc.$NAMESPACE.svc.cluster.local
-      cqlsh -e 'SHOW HOST' --cqlversion="3.4.4"
+      set -e
+      for i in $(seq 0 2); do
```

Comment:
Could we rely on `$REPLICAS` var instead?

Reply:
It depends - do you think that this test is checking wrong expected behavior?